README.md

```
@@ -1 +1,197 @@
+```bash
+npm run check     # 問題の検出
+npm run check:fix # 自動修正
+```
+
+Biome によるリントとフォーマットを実行する。
+
+---
+
+## 📁 プロジェクト構成
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)